### PR TITLE
#6| Create login page

### PR DESCRIPTION
**Pull request by kzywarash**

### CODE REVIEW

File: grails-app/controllers/groovyschool/UrlMappings.groovy [hunk 1]
```diff
@@ -13,5 +13,6 @@ class UrlMappings {
         "/register"(view:"/register")
         "500"(view:'/error')
         "404"(view:'/notFound')
+        "/login"(view:"/login")
```
Comment (kraulain) [inline grails-app/controllers/groovyschool/UrlMappings.groovy]: please, could you move this line up before the "500" line?

**Review by kraulain — CHANGES_REQUESTED** [review]

**Review by kraulain — CHANGES_REQUESTED** [review]
looks like you copied and re-used the registration page instead of the login page from the shared template (shared on google drive). please make sure to use that one instead and have your login page look like that in the picture that was attached to you ticket

File: grails-app/controllers/groovyschool/UrlMappings.groovy (hunk 1 above)
Comment (kzywarash) [reply grails-app/controllers/groovyschool/UrlMappings.groovy]: > please, could you move this line up before the "500" line?

Ok well do so immediately

File: grails-app/views/login.gsp [hunk 2]
```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta name="layout" content="main"/>
+	<title>Log In - Groovy School</title>
+</head>
+<body class="full-screen login">
```
Comment (kraulain) [inline grails-app/views/login.gsp:7]: the CSS classes in the body tag here will get overwritten by grails. reason why in the similar register.gsp i introduced the `<main class="full-screen register"> .... </main>` tag after the body

**Review by kraulain — CHANGES_REQUESTED** [review]

**Pull request merged**